K2Bridge.Tests.UnitTests/Visitors/DateHistogramAggregationVisitorTests.cs

```diff
@@ -11,14 +11,13 @@ namespace UnitTests.K2Bridge.Visitors
     [TestFixture]
     public class DateHistogramAggregationVisitorTests
     {
-        [TestCase(ExpectedResult = "wibble by _alias = wobble\n| order by _alias asc")]
+        [TestCase(ExpectedResult = "count() by ['key'] = field\n| order by ['key'] asc")]
```

Comment:
so now count() is default?
you removed the metric name

Reply:
Yes. In my opinion, Metric will be always count() for all Bucket aggregation and this metric will never change. If you request another default metric (for instance Average), a sub aggregation is requested, it does not change the default count() metric.

So I updated the property to readonly. Anyway I can rollback if you consider there is a chance it will not be the case. @tomconte

Reply:
As far as I can tell, ES bucket aggregations always include a count, so the count() will always be there and will always be a count. That said since the Metric property still exists, I would use it in the tests anyway instead of hard coding the value...

Reply:
I think this makes sense, simplifies readability, and can go to something more generic if the need arises, which is not the case today

Reply:
I discussed the point with @tomconte, and I will come back to Metric with a get/set. We can postpone this discussion later, when we'll cover all the aggregations buckets to decide if we can pass the property to readonly.